languagetool-language-modules/de/src/main/java/org/languagetool/rules/de/OldSpellingRule.java

```diff
@@ -68,12 +78,11 @@ public RuleMatch[] match(AnalyzedSentence sentence) {
       }
       boolean ignore = false;
       for (String exception : EXCEPTIONS) {
-        if (hit.begin + exception.length() <= text.length()) {
-          String excCovered = text.substring(hit.begin, hit.begin + exception.length());
-          if (excCovered.equals(exception)) {
-            ignore = true;
-            break;
-          }
+        String pattern = "(?i)\\b" + Pattern.quote(exception) + "\\b";
```

Comment:
This line/variable doesn't seem to be used

Reply:
Should I undo the changes again?

Reply:
Not needed, but you could delete this line, as it's not needed. IntelliJ should also show `pattern` in light gray, indicating the variable is never used.

Reply:
Thanks!
commit: https://github.com/languagetool-org/languagetool/pull/9967/commits/200fe554a26222f2224b2209e3d7dd9ea9e03000